backend/pkg/rest/runtime_scan_controller.go

```diff
@@ -106,10 +111,32 @@ func (s *Server) GetRuntimeScanResults(params operations.GetRuntimeScanResultsPa
 			WithPayload(oopsResponse)
 	}
 
+	cisDockerBenchmarkCounters, err := s.getRuntimeScanCisDockerBenchmarkCounters(&database.CountFilters{
+		ApplicationIDs:             state.runtimeScanApplicationIDs,
+		CisDockerBenchmarkLevelGte: params.CisDockerBenchmarkLevelGte,
+	})
+	if err != nil {
+		log.Errorf("Failed to get cis docker benchmark counters: %v", err)
+		return operations.NewGetRuntimeScanResultsDefault(http.StatusInternalServerError).
+			WithPayload(oopsResponse)
+	}
+
+	cisDockerBenchmarkCountPerLevel, err := s.dbHandler.CISDockerBenchmarkResultTable().CountPerLevel(&database.CountFilters{
+		ApplicationIDs: state.runtimeScanApplicationIDs,
+	})
+	if err != nil {
+		log.Errorf("Failed to get cis docker bencmark issues per level: %v", err)
+		return operations.NewGetRuntimeScanResultsDefault(http.StatusInternalServerError).
+			WithPayload(oopsResponse)
+	}
+
 	return operations.NewGetRuntimeScanResultsOK().WithPayload(&models.RuntimeScanResults{
-		Counters:                 counters,
-		Failures:                 failures,
-		VulnerabilityPerSeverity: vulPerSeverity,
+		CisDockerBenchmarkCountPerLevel: cisDockerBenchmarkCountPerLevel,
+		CisDockerBenchmarkCounters:      cisDockerBenchmarkCounters,
+		CisDockerBenchmarkScanEnabled:   quickScanConfig.CisDockerBenchmarkScanEnabled,
```

Comment:
it's not correct to show whether CIS benchmark was enabled in the last runtime scan by fetching it from config, because it can be set post-scan.

Reply:
It is fetched from [local state](https://github.com/cisco-open/kubei/blob/d44ddca5fd2a0d4b515a73ccea121408bff808bd/backend/pkg/rest/runtime_scan_controller.go#L74) and not from DB. The local state is being changed only when a [new startScan command arrived](https://github.com/cisco-open/kubei/blob/d44ddca5fd2a0d4b515a73ccea121408bff808bd/backend/pkg/rest/runtime_scan_controller.go#L283). So even if the config was changed post-scan it will not be changed on local state unless there is a new scan.

Reply:
oh, I see. Can you please add a comment that it's the last scan config and not the DB config?